gptcli/assistant.py

```diff
@@ -83,6 +84,8 @@ def get_completion_provider(model: str) -> CompletionProvider:
         return CohereCompletionProvider()
     elif model.startswith("gemini"):
         return GoogleCompletionProvider()
+    elif gptcli.providers.openai.use_azure:
+        return OpenAICompletionProvider()
```

Comment:
Instead of having Azure be essentially the default, I think we can add a prefix to Azure model names, similar to `oai-compat:` - maybe `oai-azure:`?